ryot_compass/src/bevy_compass/drawing/draw.rs

```diff
@@ -29,7 +31,7 @@ pub fn draw_on_click<C: ContentAssets>() -> SystemConfigs {
 fn draw_to_tile<C: ContentAssets, F: ReadOnlyWorldQuery>(
     mut commands: Commands,
     mut tiles: ResMut<MapTiles>,
-    lmdb_env: LmdbResource,
+    #[cfg(feature = "lmdb")] lmdb_env: LmdbResource,
```

Comment:
I didnt know this was possible, you can remove lmdbresouce completely and use Res<LmdbEnv>

Reply:
ah cool, done!